namespaces/cloud-platform-live-0.k8s.integration.dsd.io/pq-staging/resources/ecr.tf

```diff
@@ -1,31 +0,0 @@
-#########################################
-# Parliamentary Questions ECR (STAGING) #
-#########################################
-
-terraform {
```

Comment:
You will need to keep the `terraform` and aws `provider` blocks or this won't run at all.